app/presenters/cohorts/students_presenter.rb

```diff
@@ -144,6 +144,10 @@ def milestone_targets
         @course.targets.live.where(milestone: true).order(:milestone_number)
     end
 
+    def page_title
+      "#{@cohort.name} | #{@course.name}"
```

Comment:
Something I missed in the last review. We need to have these strings in translation file `en.yml` and pass the values as variables. You can check other pages for examples.

Reply:
I presume you meant something like this: 

```
def page_title
      I18n.t("shared.cohorts") + " | #{@course.name}"
end
```

Not applicable here as both are queried values. Added to the other file where we have one hardcoded string.
[app/presenters/courses/cohorts_presenter.rb](https://github.com/pupilfirst/pupilfirst/pull/1419/files#diff-353260ec44925da076f04f7659d2ca3685cd844c8460e5b1211522cc6c7c89a5)

Reply:
@sameer1612 Yes, you are correct that the entire string is generated from variable values. But a small nuance here is that in some languages the preferred order would be different. So having them as variables in the translation file helps. Please note the changes I have made here.